gdx/src/com/badlogic/gdx/utils/Json.java

```diff
@@ -46,6 +46,7 @@ public class Json {
 	static private final boolean debug = false;
 
 	private JsonWriter writer;
+	private JsonReader reader = new JsonReader();
```

Comment:
A `JsonReader` seems to store some states when parsing, so I'm not sure whether having one `JsonReader` per `Json` class is non-breaking.

Reply:
I thought so too. But then I ran the `JsonTest` and everything worked fine. I will check it more thoroughly (I also overlooked that there are attributes in the class because they are not at the top)

Reply:
I added a test where the reader is reused. So I think it works. Or do you have any other ideas for tests?

Reply:
JsonReader fields are only used during `parse(...)`, it's safe to use the same reader instance multiple times.